sdk-workflows/src/main/java/io/dapr/workflows/client/DaprWorkflowClient.java

```diff
@@ -262,7 +262,7 @@ public void close() throws InterruptedException {
     }
   }
 
-  private static ClientInterceptor WORKFLOW_INTERCEPTOR = new ClientInterceptor() {
+  public static ClientInterceptor WORKFLOW_INTERCEPTOR = new ClientInterceptor() {
```

Comment:
Is there a way to avoid making this public? This makes this part of the SDK user facing surface with this change.

Reply:
Because this is an interceptor, I believe it is ok to make it public and part of the interface, as this functionality can apply to other application code where the Dapr Token might want to be added.. does that make sense @artursouza ?

Reply:
I propose we create a new class, called WorkflowGrpcInterceptor (or something similar) in a packaged called io.dapr.workflows.internal because that communicate that anything in that package is for internal use and is not part of our interface with the application.

So, we can use the same class in both places here but just will not be a shared instance.

Reply:
done